app.py

```diff
@@ -246,17 +244,21 @@ def calc_data(pair, range=0.05, maxSize=32, minVolumePerc=0.01, ob_points=30):
     vol_grp_bid['max_Price'] = vol_grp_bid['max_Price'].apply(round_sig, args=(3, 0, 2))
     vol_grp_ask['max_Price'] = vol_grp_ask['max_Price'].apply(round_sig, args=(3, 0, 2))
 
-    # Append individual text to each elem
+    # Round and format the Total Price
+    vol_grp_bid['total_price'] = (vol_grp_bid['total_price'].round(2).apply(lambda x: "{:,}".format(x)))
+    vol_grp_ask['total_price'] = (vol_grp_ask['total_price'].round(2).apply(lambda x: "{:,}".format(x)))
```

Comment:
Change to `.apply(round_sig, args=(3, 0, 2))` for having 3 significant decimals for low volume pairs (LTC-BTC wich has a little total of BTC)

Reply:
These decimals aren't really important when it comes to total price, especially for *-BTC pairs, the total price is really just meaningful psychological information for USD/EUR pairs so you know how big an order really is. I doubt anyone looks at total order size in BTC. For example, 232.02 LTC for 0.0232 BTC rounded to 5.38 BTC instead of 5.382 BTC is really not an issue. That's why line 213 just uses round(2) as well.

Reply:
Okay. Maybe I personally disagree with row, cause I don´t like using lamda^^

Reply:
I used it to format the number from saying e.g. $545944.55 to $545.944,55 which is more human-readable.